### ISSUE TITLE
[CLI] cannot login

### ISSUE THREAD

**Issue by demegire** [wandb/client#2146]
**Description**
I cant login. I use the latest wandb.

I used to login with wandb.login() but that seems to be depreciated. Now I try !wandb login from my jupyter notebook but cant provide my api key as text input.

![Screenshot (462)](https://user-images.githubusercontent.com/62503047/116776856-8ee0f300-aa73-11eb-97bd-b4a44dbf60d2.png)

**Comment by ariG23498** [comment]
Hey @demegire 
Thanks for the ticket.
I reproduced the issue in a jupyter notebook with wandb version `0.10.28`. The text field does not show at all.
![image](https://user-images.githubusercontent.com/36856589/116844388-9aadf000-ac00-11eb-8f1a-131d292c454a.png)

I have also tried doing the same in a colab notebook. There things seem to be working as usual.
![image](https://user-images.githubusercontent.com/36856589/116844374-8f5ac480-ac00-11eb-99f6-25823ea4277f.png)

> Workaround: A helpful workaround would be to login at your terminal using wandb login. I can confirm that the text field works as usual in a terminal. You could also put your API key in the `wandb.login(key="XXXX")` in the notebook itself. In the meantime we will be tracking this issue internally.
